code_generator/.gitignore

```diff
@@ -328,7 +328,6 @@ ASALocalRun/
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
 
-Generated/
+/Generated/
 End2End/schema_json_ast/
-End2End/GeneratedDependencies/Generated/
```

Comment:
do we want to remove this line?

Reply:
Yes, at the request of @samiwh we check in the example generated code. That way, any changes that someone makes to the templates is easier to parse by looking at the example generated code.

Reply:
I'd argue that was the entire point of adding the tests. We could also do this by having a unit test that specifies what the generated code should look like (the original suggestion) but these seems good enough.